travis/setup_dependencies_common.sh

```diff
@@ -138,4 +137,10 @@ if [[ $ASTROPY_VERSION == dev* ]]; then
     $PIP_INSTALL git+http://github.com/astropy/astropy.git#egg=astropy --upgrade
 fi
 
+if [[ $DEBUG == True ]]; then
```

Comment:
I think yes as it only tests for extistence and not the bool value, so it's always True for any value DEBUG is set.

```
bsipocz@tegla2:~/munka/devel/ci-helpers$ a=False
bsipocz@tegla2:~/munka/devel/ci-helpers$ if [[ $a ]]; then echo 1;fi
1
bsipocz@tegla2:~/munka/devel/ci-helpers$ a=qqriq
bsipocz@tegla2:~/munka/devel/ci-helpers$ if [[ $a ]]; then echo 1;fi
1
```